lib/mfg.c

```diff
@@ -145,6 +147,42 @@ int switchtec_security_config_get(struct switchtec_dev *dev,
 	return 0;
 }
 
+/**
+ * @brief Retrieve mailbox entries
+ * @param[in]  dev	Switchtec device handle
+ * @param[in]  fd	File handle to write the log data
+ * @return 0 on success, error code on failure
+ */
+int switchtec_mailbox_get(struct switchtec_dev *dev, int fd)
```

Comment:
It might be better to name it like switchtec_mailbox_to_fd() or similar to indicate that this API eventually write the mailbox log into a file.